src/com/goide/runconfig/GoBuildingRunner.java

```diff
@@ -55,6 +56,9 @@ public String getRunnerId() {
 
   @Override
   public boolean canRun(@NotNull String executorId, @NotNull RunProfile profile) {
+    if(DefaultDebugExecutor.EXECUTOR_ID.equals(executorId)) {
```

Comment:
I should learn boolean conditions I guess :D I didn't realized that's working because the menu option for debugging is always enabled and I what was in my mind was to always disable it forcefully.